FragmentService/AbstractFragmentService.php

```diff
@@ -109,8 +120,13 @@ public function configureOptions(OptionsResolver $resolver)
     /**
      * {@inheritdoc}
      */
-    public function validate(FragmentInterface $fragment, ExecutionContextInterface $context)
+    public function validate(ErrorElement $errorElement, $object)
     {
+        if (empty($object->getBackofficeTitle())) {
```

Comment:
the whole validation part is still a work in progress, for the moment we can't use symfony constraints, we are working on that on another PR. I will add the todo in doc.